vminfo_parser/config.py

```diff
@@ -65,6 +65,12 @@ def _get_parser() -> argparse.ArgumentParser:
         default=False,
         help="Show disk space by OS",
     )
+    parser.add_argument(
+        "--disk-space-by-granular-os",
+        action="store_true",
+        default=False,
+        help="When getting disk space by os, this breaks those results more granularly",
+    )
```

Comment:
If we are going to add config flags,  we should add E2E test cases for them,  especially until we have unit tests for main and analyzer written